lib/node_modules/@stdlib/blas/ext/base/sdsnansum/README.md

```diff
@@ -139,22 +132,14 @@ var v = sdsnansum.ndarray( N, x, 2, 1 );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float32Array = require( '@stdlib/array/float32' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var sdsnansum = require( '@stdlib/blas/ext/base/sdsnansum' );
 
-var x;
-var i;
-
-x = new Float32Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-    if ( randu() < 0.2 ) {
-        x[ i ] = NaN;
-    } else {
-        x[ i ] = round( randu()*100.0 );
-    }
-}
+
+var rand = discreteUniform( 0, 100 );
+var i = filledarrayBy( len, 'float32', rand );
+var x = i.map(el => discreteUniform( 0.0, 1.0 ) < 0.2 ? el : NaN)
```

Comment:
This isn't necessary. Provide a single function to `filledarrayBy` which handles both `rand` and `NaN` value generation. Meaning, we can create a filled array in a single pass.

Reply:
Have changed